src/input.cc

```diff
@@ -292,6 +292,8 @@ void inputEventQueueReset()
 {
     gInputEventQueueReadIndex = -1;
     gInputEventQueueWriteIndex = 0;
+    SDL_Event e;
+    while (SDL_PollEvent(&e)) {} // Clear all input events
```

Comment:
```suggestion
    while (SDL_PollEvent(&e)) { } // Clear all input events
```

This should eliminate CLang warning.